conf/pacific/rgw/tier-2_rgw_multisite_archive_with_haproxy_conf.yaml

```diff
@@ -73,7 +73,7 @@ globals:
         role:
           - mon
           - osd
-
+          - rgw
```

Comment:
@viduship, could you please also update reef conf

Reply:
done.